cmd/stakepool.go

```diff
@@ -85,12 +86,13 @@ var spInfo = &cobra.Command{
 
 		doJSON, _ := cmd.Flags().GetBool("json")
 
-		if flags.Changed("blobber_id") {
-			if blobberID, err = flags.GetString("blobber_id"); err != nil {
-				log.Fatalf("can't get 'blobber_id' flag: %v", err)
-			}
+		if !flags.Changed("blobber_id") {
+			PrintError("Error: blobber_id flag is missing")
```

Comment:
```suggestion
			log.Fatalf("Error: blobber_id flag is missing")
```

`os.Exit(1)` can be removed.